static/sass/_patterns_takeunders.scss

```diff
@@ -0,0 +1,23 @@
+/// homepage takeunders
+@mixin maas-p-takeunders {
+  .p-takeunder {
+    border-radius: .125rem;
+    color: $color-x-light;
```

Comment:
Yes, I see the background is set inline but that is not as flexible as being able to switch the color if the next design has a light background. This should also have been picked up in ubuntu.com.